base/inc/AdePT/BVHNavigator.h

```diff
@@ -9,352 +9,12 @@
-    Precision step = ComputeStepAndHit(localpoint, localdir, step_limit, in_state, out_state, hitcandidate);
-    step += push;
-
-    if (out_state.IsOnBoundary()) {
-      if (!hitcandidate) {
-        vecgeom::VPlacedVolume const *currentmother       = out_state.Top();
-        Vector3D transformed                              = localpoint;
-        // Push the point inside the next volume.
-        transformed += (step + kBoundaryPush) * localdir;
-
-        do {
-          out_state.SetLastExited();
-          out_state.Pop();
-          transformed   = currentmother->GetTransformation()->InverseTransform(transformed);
-          currentmother = out_state.Top();
-        } while (currentmother && (currentmother->IsAssembly() || !currentmother->UnplacedContains(transformed)));
-      } else {
-        out_state.Push(hitcandidate);
-      }
-    }
-
-    return step;
-  }
-
-  // Computes a step from the globalpoint (which must be in the current volume)
-  // into globaldir, taking step_limit into account.
-  __host__ __device__ static Precision ComputeStepToApproachNextVolume(Vector3D const &globalpoint,
-                                                                       Vector3D const &globaldir, Precision step_limit,
-                                                                       vecgeom::NavStateIndex const &in_state)
-  {
-    // calculate local point/dir from global point/dir
-    Vector3D localpoint;
-    Vector3D localdir;
-    // Impl::DoGlobalToLocalTransformation(in_state, globalpoint, globaldir, localpoint, localdir);
-    vecgeom::Transformation3D m;
-    in_state.TopMatrix(m);
-    localpoint = m.Transform(globalpoint);
-    localdir   = m.TransformDirection(globaldir);
-
-    Precision step = ApproachNextVolume(localpoint, localdir, step_limit, in_state);
-
-    return step;
-  }
-
-  // Relocate a state that was returned from ComputeStepAndNextVolume: It
-  // recursively locates the pushed point in the containing volume.
-  __host__ __device__ static void RelocateToNextVolume(Vector3D const &globalpoint, Vector3D const &globaldir,
-                                                       vecgeom::NavStateIndex &state)
-  {
-    // Push the point inside the next volume.
-    Vector3D pushed = globalpoint + kBoundaryPush * globaldir;
-
-    // Calculate local point from global point.
-    vecgeom::Transformation3D m;
-    state.TopMatrix(m);
-    Vector3D localpoint = m.Transform(pushed);
-
-    VPlacedVolumePtr_t pvol = state.Top();
-
-    state.Pop();
-    LocatePointIn(pvol, localpoint, state, false, state.GetLastExited());
-
-    if (state.Top() != nullptr) {
-      while (state.Top()->IsAssembly()) {
-        state.Pop();
-      }
-      assert(!state.Top()->GetLogicalVolume()->GetUnplacedVolume()->IsAssembly());
-    }
-  }
-};
+using BVHNavigator = vecgeom::BVHNavigator;
 
 } // End namespace COPCORE_IMPL
-#endif // RT_LOOP_NAVIGATOR_H_
+#endif // RT_LOOP_NAVIGATOR_H_
```

Comment:
RT_NAVIGATOR_H_

Reply:
OK we fix this in a separate PR not to re-trigger the CI now